dstyle.dd

```diff
@@ -70,8 +70,8 @@ struct FooAndBar;
 
     $(DD Templates which have the same name as a symbol within that template
          (and instantiations of that template are therefore replaced with that
-         symbol) should be capitalized in the same way that that inner symbol
-         would be capitalized if it weren't in a template - e.g. types should be
+         symbol) should be capitalized in the same way that inner symbol would
+	 be capitalized if it weren't in a template - e.g. types should be
```

Comment:
That's a TAB which is rejected.

Reply:
Ups, my vim made some unwanted mistakes :) Fixed.